### ISSUE TITLE
Changing background color dosen't work , and Enhancement request to allow background color of theme

### ISSUE THREAD

**Issue by v3ss0n** [romgrk/termrk#16]
I tried this and dosen't work

```
.termrk .terminal {
    background-color:  @app-background-color
}
```

**Comment by v3ss0n** [comment]
this one works

```
atom-panel.termrk-panel .termrk-container {
    background-color:  @app-background-color
}
```

**Comment by v3ss0n** [comment]
it fails to work again after atom restart.

**Comment by v3ss0n** [comment]
I have to do this to ovveride background color and make it transparent.

``` css
atom-panel.termrk-panel .termrk-container{
    background-color:  @app-background-color;
    border: 5px solid transparent;
}
atom-panel .termrk .terminal {
    background-color: @app-background-color;
}
```

**Comment by romgrk** [comment]
Well noted. Working on a fix, the styling format might change.

**Comment by romgrk** [comment]
Fixed 1ddc5b6047ba4283a63ac894fa77b27f51f319cc

Styling should now work as described in README.md, using `.termrk .terminal` and `.termrk .terminal-cursor`.
